compiler/onecc-docker/debian/control

```diff
@@ -0,0 +1,13 @@
+Source: onecc-docker
+Section: devel
+Priority: extra
+Maintainer: Neural Network Acceleration Solution Developers <nnfw@samsung.com>
+Build-Depends: debhelper (>=9)
+Standards-Version: 4.5.1
+Homepage: https://github.com/Samsung/ONE
+
+Package: onecc-docker
+Architecture: amd64
+Multi-Arch: foreign
+Depends: ${misc:Depends}, ${shlibs:Depends}, docker, python3
```

Comment:
```suggestion
Depends: ${misc:Depends}, ${shlibs:Depends}, python3.8
```

Reply:
@jyoungyun 

Oh, thank you so much for the comment! But I have a question. Do we remove dependence on this docker because we assume that it is installed on the docker before installing onecc-docker debian package?

Reply:
Is it installed normally well if we specify the docker as depends?
In order to install the docker engine, several processes are required, so I think the docker is not installed simply by adding dependence.

Thats' why we wrote in the requirements that we need a docker in the README.md file. :)